spec/support/features/session_helper.rb

```diff
@@ -162,7 +162,7 @@ def acknowledge_and_confirm_recovery_code
       click_on button_text, class: 'recovery-code-continue'
 
       code_words.size.times do |index|
-        fill_in "recovery-#{index}", with: code_words[index].downcase
+        fill_in "recovery-#{index}", with: code_words[index].downcase + 'xxxx'
```

Comment:
Oh, right. When validating input, we don't check, but we probably should according to that issue, to reduce our attack surface. Since we know a password can only be between 8 and 128 characters, we can outright reject any that are outside of that range.